benchmarks/newton_solver_benchmark_set/nonlinear_channel_flow/README.md

```diff
@@ -0,0 +1 @@
+The files in this directory can be used to recreate the nonlinear channel flow figures from the Fraters et al (subm) paper. Adjust both the metabash.sh and bash.sh files to reflect the parameter search you are interested in and run the metabash script. If you have run exactly the same runs as shown in the paper, you can use the gnuplot files in plotfiles to recreate the figures in that paper. This benchmark requires the normal nonlinear channeflow benchmark to be compiled. 
```

Comment:
et al. (with a period).

Instead of subm, I think we're certain enough that this will be published eventually -- so say "G-Cubed, submitted, 2018" to make it easier for people later on to find the paper in case we forget to update it here. In fact, maybe list all authors just for good measure.